bucket/juce.json

```
@@ -0,0 +1,22 @@
+{
+    "version": "5.4.7",
+    "description": "Open-source cross-platform C++ audio application framework",
+    "homepage": "https://juce.com/",
+    "license": {
+        "identifier": "ISC,GPL-3.0-or-later",
+        "url": "https://juce.com/juce-5-license"
+    },    
+    "url": "https://github.com/juce-framework/JUCE/releases/download/5.4.7/juce-5.4.7-windows.zip",
+    "extract_dir": "JUCE",
+    "bin": "Projucer.exe",
+    "shortcuts": [
+        [
+            "Projucer.exe",
+            "Projucer"
+        ]
+    ],
+    "checkver": "github",
```

Homepage is not github repository ....

```suggestion
    "checkver": {
        "github": "https://github.com/juce-framework/JUCE"
    },
```